aggregator/config/registries.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "url": "http://localhost:9090/services",
+        "url": "http://192.168.1.15:9050/services",
```

Comment:
please don't modify default config, this is deployment specific and can be overwritten